client/src/components/geneExpression/gene.js

```diff
@@ -0,0 +1,191 @@
+              justifyContent: "space-between",
+            }}
+          >
+            <Icon
+              icon="drag-handle-horizontal"
+              iconSize={12}
+              style={{
+                marginRight: 7,
+                cursor: "grab",
+                position: "relative",
+                top: 3,
+              }}
+            />
+            <Truncate>
+              <span
+                style={{
+                  width:
+                    globals.leftSidebarWidth -
+                    310 /* todo_genesets this magic number determines how much of a long geneset name we see, and will be tweaked as we build */,
+                }}
+                data-testid={`${gene}:gene-label`}
+              >
+                {gene}
+              </span>
+            </Truncate>
+            {!geneIsExpanded ? (
+              <HistogramBrush isUserDefined field={gene} mini />
+            ) : null}
+          </span>
+          <span>
+            {!isDiffexp ? (
+              <AnchorButton
+                minimal
+                small
+                data-testid={`delete-from-geneset-${gene}`}
+                onClick={this.handleDeleteGeneFromSet}
+                intent="none"
+                style={{ fontWeight: 700, marginRight: 2 }}
+                icon={<Icon icon="trash" iconSize={10} />}
+              />
+            ) : null}
+            <AnchorButton
+              minimal
+              small
+              data-testid={`plot-x-${gene}`}
+              onClick={this.handleSetGeneAsScatterplotX}
+              active={isScatterplotXXaccessor}
+              intent={isScatterplotXXaccessor ? "primary" : "none"}
+              style={{ fontWeight: 700, marginRight: 2 }}
+            >
+              x
+            </AnchorButton>
+            <AnchorButton
+              minimal
+              small
+              data-testid={`plot-y-${gene}`}
+              onClick={this.handleSetGeneAsScatterplotY}
+              active={isScatterplotYYaccessor}
+              intent={isScatterplotYYaccessor ? "primary" : "none"}
+              style={{ fontWeight: 700, marginRight: 2 }}
+            >
+              y
+            </AnchorButton>
+            <AnchorButton
+              minimal
+              small
+              data-testclass="maximize"
+              data-testid={`maximize-${gene}`}
+              onClick={this.handleGeneExpandClick}
+              active={geneIsExpanded}
+              intent="none"
+              icon={<Icon icon="maximize" iconSize={10} />}
+              style={{ marginRight: 2 }}
+            />
+            <AnchorButton
+              minimal
+              small
+              data-testclass="colorby"
+              data-testid={`colorby-${gene}`}
+              onClick={this.onColorChangeClick}
+              active={isColorAccessor}
+              intent={isColorAccessor ? "primary" : "none"}
+              icon={<Icon icon="tint" iconSize={12} />}
```

Comment:
IconName constant

Reply:
I'll make an issue, we do this everywhere